llvm/lib/Target/RISCV/RISCVInstrInfo.td

```diff
@@ -2032,6 +2032,28 @@ def : Pat<(binop_allwusers<add> GPR:$rs1, (AddiPair:$rs2)),
                  (AddiPairImmSmall AddiPair:$rs2))>;
 }
 
+//===----------------------------------------------------------------------===//
+// GlobalISel generic instruction definitions
+/* /===----------------------------------------------------------------------===//
+
+class RISCVGenericInstruction : GenericInstruction {
+  let Namespace = "RISCV";
+}
+
+def G_ADD_LO : RISCVGenericInstruction {
```

Comment:
G_RISCV_?

Reply:
I abandoned using these generic instructions. If you look at the next comment, you'll see they were deleted; I didn't squash the comment so that I could preserve that version in case we decided to revert to that.

In the future, would it be better if I squashed such changed?

Reply:
If we decide to go with the generic instruction approach, I will rename them accordingly, though, it does make things much clearer.

Reply:
> I abandoned using these generic instructions

These look to be part of the overall diff still. Do you plan on removing the changes in this file if you are not going to use them?